### PR TITLE
feat: added support for sale price

### PR DESCRIPTION
**Pull request by wwalser**
feat: added support for sale price

### CODE REVIEW

File: .changeset/odd-yaks-look.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'sajari-sdk-docs': minor
+'@sajari/react-search-ui': minor
+---
+
+feat: added support for sale price
```
Comment (zlatanpham) [inline .changeset/odd-yaks-look.md:6]: We normally don't include a commit type in a changeset file. Can you trim it out?

```suggestion
Added support for sale price
```

**Review by zlatanpham — DISMISSED** [review]
I just left a minor comment on the changeset format message, otherwise, the code looks good to me.

**Comment by zlatanpham** [comment]
@tuanddd you might want to take a look too to see if the change works with your recent `variant` change.

**Comment by tuanddd** [comment]
> @tuanddd you might want to take a look too to see if the change works with your recent `variant` change.

Currently checking it out.

File: packages/search-ui/src/Results/components/Result/index.tsx [hunk 2]
```diff
@@ -133,35 +143,47 @@ const Result = React.memo(
...
+        // show `originalPrice` with strikethrough
+        const activeOriginalPrice = isArray(originalPrice)
+          ? originalPrice[activeImageIndex] ?? originalPrice
+          : originalPrice ?? '';
+        priceToDisplay = formatPrice(activePrice, { currency, language });
+        markedDownFromPriceToDisplay = formatPrice(activeOriginalPrice, { currency, language });
+      } else if (salePrice && isOnSale) {
+        // show `price` with strikethrough
+        const activeSalePrice = isArray(salePrice) ? salePrice[activeImageIndex] ?? salePrice : salePrice ?? '';
+        if (activeSalePrice !== '0') {
```
Comment (zlatanpham) [inline packages/search-ui/src/Results/components/Result/index.tsx:161]: I left a question in slack about the necessity of the `salePrice` if we decide to check against `'0'`.

Comment (zlatanpham) [reply packages/search-ui/src/Results/components/Result/index.tsx:161]: Got the answer in slack

**Comment by tuanddd** [comment]
LGTM

**Pull request merged**